### ISSUE TITLE
About pre-trained  model of facenet

### ISSUE THREAD

**Issue by junyizeng** [jinyugy21/Adv-Stickers_RHDE#3]
Thank you very much for your open source your work.The link you provided to fanet-related resources does not seem to provide net_13_022.pth, how can I get this file, or can you provide it?
![image](https://user-images.githubusercontent.com/80525097/211301041-eafa44ca-b1b5-4e08-98ce-a2afd2118719.png)

**Comment by junyizeng** [comment]
Could you provide the code for the face recognition model fine-tune or the model file after fine-tune？

![image](https://user-images.githubusercontent.com/80525097/215313871-ceef78bd-2505-4a06-a4cc-14ad10050f47.png)

**Comment by jinyugy21** [comment]
> Could you provide the code for the face recognition model fine-tune or the model file after fine-tune？
> 
> ![image](https://user-images.githubusercontent.com/80525097/215313871-ceef78bd-2505-4a06-a4cc-14ad10050f47.png)

Sorry for the late reply due to the Spring Festival.
Our experiment is carried out on the closed-set classification task. We use an open-source feature extractor and train a classifier based on our own dataset, and the two together form a model. Since my and your datasets may contain different kinds of identities, our classifier may not generalize to your data.
But it is easy to train a classifier after fixing the feature extractor, you can refer to links below:
https://github.com/ronghuaiyang/arcface-pytorch/blob/master/train.py
https://github.com/timesler/facenet-pytorch/blob/8d09930ffb21676a9187267579fb697f67c1b934/examples/finetune.ipynb

**Comment by junyizeng** [comment]
Thank you for your reply. Can you provide some specific parameters of your finetune? (For example, the loss of fine-tune, fine-tune step size, learning rate, optimizer parameters) In addition, other stickers are used in your paper. Could you provide the sticker data set you used? Or the way to get stickers.
